controllers/tests/array_action/svc/array_mediator_svc_test.py

```diff
@@ -99,6 +105,200 @@ def test_close(self):
         self.svc.disconnect()
         self.svc.client.close.assert_called_with()
 
+    def test_get_replication_success(self):
+        replication_request = ReplicationRequest(OBJECT_INTERNAL_ID, OTHER_OBJECT_INTERNAL_ID, SYSTEM_ID, COPY_TYPE,
+                                                 REPLICATION_TYPE_MIRROR)
+        rcrelationships_to_return = \
+            [Munch({svc_settings.RCRELATIONSHIP_STATE_ATTR_NAME: RCRELATIONSHIP_STATE_READY,
+                    svc_settings.RCRELATIONSHIP_COPY_TYPE_ATTR_NAME: svc_settings.RCRELATIONSHIP_COPY_TYPE,
+                    svc_settings.RCRELATIONSHIP_MASTER_CLUSTER_ID_ATTR_NAME: "",
+                    svc_settings.RCRELATIONSHIP_PRIMARY_ATTR_NAME: False,
+                    svc_settings.RCRELATIONSHIP_ID_ATTR_NAME: svc_settings.DUMMY_INTERNAL_ID1,
+                    svc_settings.RCRELATIONSHIP_NAME_ATTR_NAME: ""})]
```

Comment:
[same](https://github.com/IBM/ibm-block-csi-driver/pull/593/files#r1020859652)

Reply:
Duplications moved to function preparing mocks

Reply:
cant see it, still see dups

Reply:
Done